tests/rptest/tests/cloud_topics/e2e_test.py

```diff
@@ -130,20 +145,20 @@ class EndToEndCloudTopicsTxTest(EndToEndCloudTopicsBase):
             replication_factor=3,
         ),
     )
+    kgo_producer: KgoVerifierProducer
+    kgo_consumer: KgoVerifierSeqConsumer
```

Comment:
These are typing fixes I'm seeing for the first time (trying out a new typechecker).